.azurepipelines/build-release-packages-1es.yml

```diff
@@ -200,6 +216,12 @@ extends:
           inputs:
             secureFile: 'appcenter-pair-key.snk'
 
+        - task: UseDotNet@2
```

Comment:
why do we need this task?

Reply:
I checked and removed it as unnecessary.